content/docs/user-guide/gems/reference/aws/aws-core/configuring-credentials.md

```diff
@@ -98,8 +126,77 @@ aws configure list
 aws configure list-profiles
 ```
 
+## Controlling Permissions for Users
+All AWS IAM users are given permissions through the attachment of managed or inline IAM polices. See [Changing permissions for an IAM user](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_change-permissions.html).
+
+Its recommended that you:
+* Use an Admin user, that is a user with full permissions on an AWS account, when absolutely necessary, such as deploying and provisioning AWS resources.
+* Grant [least privileges](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege) on your AWS users, that is they only have the minimum set of permissions on specific resources they require. This is controlled through setting of managed IAM policies and other mechanisms.
+* As you add and review AWS resources, you update managed policies to grant or revoke the appropriate permissions.
+
+### Create an IAM User Groups
+To make managing permissions easier, its recommended that you create [user groups](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_groups.html). User groups let you specify permissions for multiple users at time, which can make it easier to manage the permissions for those users.
+
+You can create user groups via the console, the cli or through the CloudFormation/CDK. We recommend you create at least two user groups to work with AWS in O3DE:
+* Admins - Administrator users required to own and manage AWS resources. Typically, performing updates and management of key resources.
+* Users - set of users that take action on the resources as part of normally gameplay or simulation.
+
+#### Create a user group via the AWS Console
+1. Sign in to the AWS Management Console, if you are not already signed in, at https://console.aws.amazon.com/iamv2
+2. Open the IAM console. To do this, in the AWS navigation bar, choose Services. Then choose **IAM**.
+3. In the IAM console's navigation pane, choose **Groups**.
+4. Choose **Create New Group**.
+5. On the Set Group Name page, for Group Name, enter a name for the new group. 
+6. Choose Next Step.
+7. On the **Attach Policy** page, choose **Next Step** without attaching any policies or attach any current policies relevant to the group.
+8. Choose **Create Group**. 
+
+#### Create a user group via the AWS CLI
+1. Install and configure the AWS CLI as above, see [Installing the AWS Command Line Interface](https://docs.aws.amazon.com/cli/latest/userguide/installing.html).
+2. Generate a user group
+```
+aws iam create-group --group-name MyUserGroup
+```
+
+#### Create user groups via the AWS CDK Examples
+The AWSCore CDK stack will autogenerate sample Admins and Users user groups for you. By default, these groups are named ```O3DE-AWS-PROJECT-Admins``` and ```O3DE-AWS-PROJECT-Users```.
+
```

Comment:
should call out even core cdk create user group, but no aws gem is using it (the connection is missing based on our current cdk setup) developer needs to fill the gap manually either through aws console or cdk.

same to adding permission to user group, it is doable to add other aws gem required permission to core created user group. (connection is missing)

Reply:
Good catch, I meant to add line about how the other Gems create policies that you can manually attach to these groups. Will add. We have future roadmap story to aid automation of permissions so its not disconnected.